### PR TITLE
Workflow fixes for better operator controls

### PR DESCRIPTION
**Pull request by jperez999**
This tackles #229 and #212

### CODE REVIEW

**Comment by jperez999** [comment]
rerun tests

File: tests/unit/test_dask_nvt.py [hunk 1]
```diff
@@ -24,8 +24,8 @@
 import pytest
 from dask.dataframe import assert_eq
 
-import nvtabular.ops as ops
 from nvtabular import Dataset, Workflow
+from nvtabular import ops as ops
```
Comment (benfred) [inline tests/unit/test_dask_nvt.py:28]:
```suggestion
from nvtabular import ops
```

File: tests/unit/test_io.py [hunk 2]
```diff
@@ -27,7 +27,7 @@
 
 import nvtabular as nvt
 import nvtabular.io
-import nvtabular.ops as ops
+from nvtabular import ops as ops
```
Comment (benfred) [inline tests/unit/test_io.py:30]:
```suggestion
from nvtabular import ops
```

File: tests/unit/test_ops.py [hunk 3]
```diff
@@ -25,7 +25,7 @@
 
 import nvtabular as nvt
 import nvtabular.io
-import nvtabular.ops as ops
+from nvtabular import ops as ops
```
Comment (benfred) [inline tests/unit/test_ops.py:28]:
```suggestion
from nvtabular import ops
```

File: tests/unit/test_tf_dataloader.py [hunk 4]
```diff
@@ -18,7 +18,7 @@
 
 import nvtabular as nvt
 import nvtabular.io
-import nvtabular.ops as ops
+from nvtabular import ops as ops
```
Comment (benfred) [inline tests/unit/test_tf_dataloader.py:21]:
```suggestion
from nvtabular import ops
```

File: tests/unit/test_workflow.py [hunk 5]
```diff
@@ -26,7 +26,7 @@
 from pandas.api.types import is_integer_dtype
 
 import nvtabular as nvt
-import nvtabular.ops as ops
+from nvtabular import ops as ops
```
Comment (benfred) [inline tests/unit/test_workflow.py:29]:
```suggestion
from nvtabular import ops
```

File: nvtabular/io.py [hunk 6]
```diff
@@ -48,6 +46,8 @@
 from dask.utils import natural_sort_key, parse_bytes
 from fsspec.core import get_fs_token_paths
 from fsspec.utils import stringify_path
+from numba import cuda as cuda
```
Comment (benfred) [inline nvtabular/io.py:49]:
```suggestion
from numba import cuda
```

File: tests/unit/test_torch_dataloader.py [hunk 7]
```diff
@@ -22,7 +22,7 @@
 from cudf.tests.utils import assert_eq
 
 import nvtabular as nvt
-import nvtabular.ops as ops
+from nvtabular import ops as ops
```
Comment (benfred) [inline tests/unit/test_torch_dataloader.py:25]:
```suggestion
from nvtabular import ops
```

**Pull request merged**